src/aind_data_transfer/jobs/s3_upload_job.py

```diff
@@ -276,7 +291,100 @@ def run_job(self) -> None:
         self.trigger_codeocean_capsule()
 
 
+class GenericS3UploadJobs:
```

Comment:
Can we call this `GenericS3UPloadJobList` or something? The extra `s` isn't obvious.